tests/test_policy.py

```diff
@@ -14,3 +16,5 @@ def test_policy_checks_glibc():
     assert policy > POLICY_PRIORITY_LOWEST
     policy = versioned_symbols_policy({"some_library.so": {"GLIBC_999"}})
     assert policy == POLICY_PRIORITY_LOWEST
+    policy = versioned_symbols_policy({"some_library.so": {"OPENSSL_1_1_0"}})
+    assert policy == POLICY_PRIORITY_HIGHEST
```

Comment:
This will in fact fail since the assignment assumes 2 values. Is it possible for a symbol to be entirely unversioned? I guess the answer to that is likely to be yes?